internal/services/cdn/cdn_frontdoor_custom_domain_resource.go

```diff
@@ -428,31 +291,11 @@ func resourceCdnFrontDoorCustomDomainDelete(d *pluginsdk.ResourceData, meta inte
 	ctx, cancel := timeouts.ForDelete(meta.(*clients.Client).StopContext, d)
 	defer cancel()
 
-	id, err := parse.FrontDoorCustomDomainID(d.Id())
+	id, err := parse.FrontDoorCustomDomainIDInsensitively(d.Id())
```

Comment:
this is coming from the Terraform Resource ID and so should be being parsed case-sensitively:

```suggestion
	id, err := parse.FrontDoorCustomDomainID(d.Id())
```